patches/v8/fix_build_deprecated_attirbute_for_older_msvc_versions.patch

```diff
@@ -0,0 +1,43 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Deepak Mohan <hop2deep@gmail.com>
+Date: Tue, 28 Jan 2020 15:48:03 -0800
+Subject: fix: usage of c++ [[deprecated]] attirbute for older msvc versions
+
+VS 2015 update 3 has a bug where [[deprecated]] attribute cannot
+be applied to constructor declarations, this is fixed in 2017 and
+higher versions, but native module compiling with this version
+will have an issue.
+
+diff --git a/include/v8config.h b/include/v8config.h
+index 40d23c35c186e4def1fcf59c28527de292d9fd7a..7d1ef7b5bf0ff6693b8a04afc1e319219c14e230 100644
+--- a/include/v8config.h
++++ b/include/v8config.h
+@@ -388,10 +388,13 @@
+ # define V8_NOINLINE /* NOT SUPPORTED */
+ #endif
+ 
+-
+ // A macro (V8_DEPRECATED) to mark classes or functions as deprecated.
+ #if defined(V8_DEPRECATION_WARNINGS)
+-# define V8_DEPRECATED(message) [[deprecated(message)]]
++# if defined(_MSC_VER) && _MSC_VER <= 1900
```

Comment:
The version number is extracted from https://devblogs.microsoft.com/cppblog/side-by-side-minor-version-msvc-toolsets-in-visual-studio-2017/